api/src/pcapi/core/subscription/dms/api.py

```diff
@@ -65,6 +64,39 @@ def try_dms_orphan_adoption(user: users_models.User) -> None:
         pcapi_repository.repository.delete(dms_orphan)
 
 
+def _has_application_update(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Naming
`_has_application_been_updated`
ou
`is_fraud_check_up_to_date`